oktapam/fwprovider/resource_server_checkout_settings.go

```diff
@@ -0,0 +1,277 @@
+			},
+			"exclude_list": schema.ListAttribute{
+				ElementType:         types.StringType,
+				Optional:            true,
+				Description:         "If provided, only the account identifiers listed are excluded from the checkout requirement. This list is only considered if `checkout_required` is set to `true`. Only one of `include_list` and `exclude_list` can be specified in a request since they are mutually exclusive.",
+				MarkdownDescription: "If provided, only the account identifiers listed are excluded from the checkout requirement. This list is only considered if `checkout_required` is set to `true`. Only one of `include_list` and `exclude_list` can be specified in a request since they are mutually exclusive.",
+			},
+			"include_list": schema.ListAttribute{
+				ElementType:         types.StringType,
+				Optional:            true,
+				Description:         "If provided, only the account identifiers listed are required to perform a checkout to access the resource. This list is only considered if `checkout_required` is set to `true`. Only one of `include_list` and `exclude_list` can be specified in a request since they are mutually exclusive.",
+				MarkdownDescription: "If provided, only the account identifiers listed are required to perform a checkout to access the resource. This list is only considered if `checkout_required` is set to `true`. Only one of `include_list` and `exclude_list` can be specified in a request since they are mutually exclusive.",
+			},
+			"project": schema.StringAttribute{
+				Required:            true,
+				Description:         "The UUID of a Project",
+				MarkdownDescription: "The UUID of a Project",
+			},
+			"resource_group": schema.StringAttribute{
+				Required:            true,
+				Description:         "The UUID of a Resource Group",
+				MarkdownDescription: "The UUID of a Resource Group",
+			},
+		},
+	}
+}
+
+func (r *serverCheckoutSettingsResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	// Retrieve the server checkout settings values from the plan
+	var plan serverCheckoutSettingsResourceModel
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Call the SDK client to creat the server checkout settings from plan
+	serverCheckoutSettings := &pam.ResourceCheckoutSettings{
+		CheckoutRequired:          plan.CheckoutRequired,
+		CheckoutDurationInSeconds: plan.CheckoutDurationInSeconds,
+		IncludeList:               plan.IncludeList,
+		ExcludeList:               plan.ExcludeList,
+	}
+
+	_, err := r.client.SDKClient.ProjectsAPI.UpdateResourceGroupServerBasedProjectCheckoutSettings(ctx, r.client.Team, plan.ResourceGroup, plan.Project).ResourceCheckoutSettings(*serverCheckoutSettings).Execute()
```

Comment:
Create a separate file under oktapam/client to call SDK and handle errors there.